src/main/resources/sam.conf

```diff
@@ -184,6 +184,10 @@ admin {
 azureServices {
   azureEnabled = ${?AZURE_ENABLED}
   allowManagedIdentityUserCreation = ${?AZURE_ALLOW_MANAGED_IDENTITY_USER_CREATION}
+  azureControlPlaneEnabled = ${?AZURE_CONTROL_PLANE_ENABLED}
```

Comment:
This controls specifically whether to use the service catalog or partner center managed resource groups. The name of this config setting should reflect that.